_blog.yml

```
@@ -506,4 +506,13 @@
   thumbnail: /blog/assets/52_bert_101/thumbnail.jpg
   date: March 2, 2022
   tags:
-  - guide
\ No newline at end of file
+  - guide
+
+- local: fine-tune-segformer
+  title: "Fine-Tune an Image Segmentation Model with a Custom Dataset"
```

I'd advocate to make this blog post about semantic segmentation, as we will have panoptic (and perhaps instance) in the future.
```suggestion
  title: "Fine-Tune a Semantic Segmentation Model with a Custom Dataset"
```